mne/transforms.py

```diff
@@ -77,8 +79,18 @@ class Transform(dict):
     ----------
     fro : str | int
         The starting coordinate frame.
+        Can be 'unknown', 'MEG device', 'isotrak', 'hpi', 'head',
```

Comment:
actually we realized this is incorrect. @fmamashli remember to fix these. It should be `'meg'` not `'MEG device'` and so on